hydradx_update/tests/test_reweighting_amm.py

```diff
@@ -186,12 +186,12 @@ def test_remove_risk_liquidity(old_state):
     delta_q = new_agents[LP_id]['q'] - old_agents[LP_id]['q']
     assert delta_q >= 0 or delta_q == pytest.approx(0)
     assert delta_r >= 0 or delta_r == pytest.approx(0)
+   
+<<<<<<< HEAD
```

Comment:
These should be removed right? these are inserted by "diff" comparison in github?